lib/streamlit/elements/image.py

```diff
@@ -386,6 +386,62 @@ def image_to_url(
         return ""
 
 
+def _check_if_svg_can_be_rendered_as_img(image: str) -> bool:
+    """SVG needs to has either viewBox or width provided to make it possible to render it as img tag"""
+    if "xlink" in image:
+        """
```

Comment:
nit: I think it is better to use `#` also for multiline comments. The docstring notation with `"""` for multiline comments works fine, but I think it is not really preferred (I don't think it is used for an inline comment anywhere in our codebase).